metals/src/main/scala/scala/meta/internal/metals/JvmOpts.scala

```diff
@@ -14,21 +14,43 @@ import scala.meta.io.AbsolutePath
  */
 object JvmOpts {
 
-  def fromWorkspaceOrEnv(workspace: AbsolutePath): List[String] = {
-    val jvmOpts = workspace.resolve(".jvmopts")
-    if (jvmOpts.isFile && Files.isReadable(jvmOpts.toNIO)) {
-      val text = FileIO.slurp(jvmOpts, StandardCharsets.UTF_8)
-      text.linesIterator.map(_.trim).filter(_.startsWith("-")).toList
+  /**
+   * Tries to get jvmopts needed for running tests, which is now not possible to do with test explorer.
+   * It will also try to use .jvmopts, filter out any -X options since they proved to be problematic,
+   */
+  def fromWorkspaceOrEnvForTest(workspace: AbsolutePath): List[String] = {
+    val forTests =
+      fromWorkspaceOrEnv(workspace, ".test-jvmopts", "TEST_JVM_OPTS")
+    if (forTests.isEmpty) {
+      fromWorkspaceOrEnv(workspace).filterNot(_.startsWith("-X"))
     } else {
-      fromEnvironment
+      forTests
     }
   }
 
-  def fromEnvironment: List[String] = {
-    Option(System.getenv("JVM_OPTS")) match {
+  def fromWorkspaceOrEnv(workspace: AbsolutePath): List[String] = {
+    fromWorkspaceOrEnv(workspace, ".jvmopts", "JVM_OPTS")
+  }
```

Comment:
This method definition seems a bit redundant. Can't you either just pass the args or add defaults to `fromWorkspaceOrEnv`?